autocomplete.R

```diff
@@ -0,0 +1,216 @@
+# Challenge 2: Autocomplete me
+# Lynn Kaack
+
+
+# The read.terms function------
+
+# I needed to make sure that names with spaces and special characters are accepted
+read.terms <- function(input.file){
+  word.data <- read.table(input.file, sep = "\t", header = TRUE, row.names = NULL, 
+                          strip.white=TRUE, quote = NULL, stringsAsFactors=F)
+  names(word.data) <- c("weight", "term")
+  # We also sort the data by term
+  sorted.words <- word.data[order(word.data$term),] 
+  sorted.words$weight <- as.numeric(sorted.words$weight)
+  return(sorted.words)
+}
+
+
+# Other functions used by autocomplete-----
+
+# A function that sorts by weight
+  # Input: A a word data frame
+  # Output: The same data frame sorted by the weight column
+sort_byWeight <- function(input.words){
```

Comment:
Better: `sort_by_weight`. I would not mix different casing styles in one function name.